scripts/remove/nginx.sh

```diff
@@ -2,24 +2,20 @@
 
 systemctl stop nginx
 
-APT='nginx-extras ssl-cert php7.0 php7.0-cli php7.0-fpm php7.0-dev php7.0-xml php7.0-curl php7.0-xmlrpc php7.0-json php7.0-mcrypt php7.0-opcache php-geoip php-xml php php-cli php-fpm php-dev php-xml php-curl php-xmlrpc php-json php-mcrypt php-opcache'
+APT='nginx-extras nginx libnginx-mod-http-fancyindex ssl-cert php7.0 php7.0-cli php7.0-fpm php7.0-dev php7.0-xml php7.0-curl php7.0-xmlrpc php7.0-json php7.0-mcrypt php7.0-opcache php-geoip php-xml php php-cli php-fpm php-dev php-xml php-curl php-xmlrpc php-json php-mcrypt php-opcache'
 for depends in $APT; do
 apt-get -qq -y --yes --force-yes remove "$depends" >/dev/null 2>&1
 done
 
-apt-get -y -q purge nginx-* php7.0-* >/dev/null 2>&1
+apt-get -y -q purge nginx-* php7.0-* php-* >/dev/null 2>&1
+apt-get -y -q autoremove >/dev/null 2>&1
 
 rm -rf /etc/nginx
 rm -rf /etc/php
 
-if [[ -d /srv/panel ]]; then
-    rm -rf /srv/panel
-    /etc/sudoers.d/panel
-    rm /etc/cron.d/set_interface
-fi
-
 if [[ -d /srv/rutorrent ]]; then
     rm -rf /srv/rutorrent
+    rm /install/.rutorrent.lock
 fi
```

Comment:
Fair point. Rapid leech can be done easily; however, I'm concerned applications like nextcloud could have a lot of data associated with it that the user may regret deleting if they didn't realize. That said, it's probable NC would be left in a broken state if the user were to reinstall nginx, so not exactly sure what to do there since removal of user data seems troublesome

Reply:
I'd deal with this later but maybe we can check if any of the apps that depend on nginx are installed, tell the user about it and then cancel. something for later, don't think anyone will be upgrading to 20.04 and then deleting nginx...